js/sum.js

```diff
@@ -0,0 +1,47 @@
+window.addEventListener('DOMContentLoaded', () => {
+    let addBtn = document.getElementById('add-btn');
+    let newNum = document.getElementById('new-num');
+    let numsContainer = document.getElementById('added-nums-container');
+    let sumElem = document.getElementById('sum');
+    let isOdd = false;
+
+    addBtn.addEventListener('click', () => {
+        if (newNum.value === '') {
+            alert('Enter a number, please!');
+            return;
+        }
+
+        isOdd = !isOdd;
+        let currentElem = document.createElement('div');
+        isOdd ? currentElem.setAttribute('class', 'current-added-elem odd') : currentElem.setAttribute('class', 'current-added-elem');
+        let addedNum = document.createElement('span');
+        let deleteBtn = document.createElement('button');
+        deleteBtn.style.all = 'unset';
```

Comment:
What is the reason for doing this?

Reply:
Тhe reason is that I want odd elements to have an extra class to add background. I create the children of this element and remove the button's default style because it does not look like a button. What would you recommend as an alternative?

Reply:
okay so you dont need this one: 'I want odd elements to have an extra class to add background.'
As I mentioned in one of my other comments you can achieve this effect only with css.
Take a look at :nth-child(odd).
Removing styles and adding additional ones on the delete button can again be done only with css. In that way you will clear a bit your code and make it a bit more readable.